code/game/area/Space Station 13 areas.dm

```diff
@@ -1867,76 +1867,6 @@ GLOBAL_LIST_EMPTY(ghostteleportlocs)
 
 //HALF-BUILT STATION (REPLACES DERELICT IN BAYCODE, ABOVE IS LEFT FOR DOWNSTREAM)
```

Comment:
Remove this comment, it was never correct (we still had derelict), and the areas it references are gone.